source/_docs/z-wave.markdown

```diff
@@ -180,6 +180,36 @@ automation:
         scene_id: 11
 ```
 
+Some devices (like the HomeSeer wall switches) allow you to do things like double, and triple click the up and down buttons and fire an event.  This does support the OpenZwave CentralScene handler.  This is an example of double clicking the on/up button:
```

Comment:
Rather than "This does support the OpenZwave CentralScene handler.", I'd prefer "These devices will also send `scene_data` to differentiate the events." The users aren't too concerned with the underlying command class.